common/src/main/kotlin/org/bibletranslationtools/maui/common/usecases/ParseFileName.kt

```diff
@@ -69,6 +69,19 @@ class ParseFileName(private val file: File) {
         }
     }
 
+    fun read(): FileData {
```

Comment:
this will replace parse(): Single<FileData>
I'm working on updating the related code usage of this